docs/dqx/docs/guide/data_profiling.mdx

```diff
@@ -362,9 +535,16 @@ The profiler supports extensive configuration options to customize the profiling
 
       # Value rounding
       "round": True,                # Round min/max values for cleaner rules
-
+                                                                 
       # Filter Options
       "filter": None,               # No filter (SQL expression)
+
+      # LLM-based Primary Key Detection options
+      "enable_llm_pk_detection": True,  # Enable LLM-based primary key detection
+      "llm_pk_detection": True,         # Another alternative to enable LLM PK detection
+      "llm_pk_detection_endpoint": "databricks-meta-llama-3-1-8b-instruct",  # LLM endpoint
+      "llm_pk_validate_duplicates": True,   # Validate detected primary keys for duplicates
+      "llm_pk_max_retries": 3,             # Maximum retries for LLM prediction
```

Comment:
I would hardcode this as 3 and remove this param. I don't think users would ever change this.